docs/pages/upgrading/self-hosted-linux.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Upgrade Self-Hosted Teleport Clusters on Linux
+description: Provides instructions for upgrading self-hosted Teleport clusters that run on Linux servers. 
+---
+
+This guide explains how to upgrade self-hosted Teleport clusters running on
+Linux servers.
+
+## Prerequisites
+
+- Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
+  which describes the sequence in which to upgrade components of your cluster.
+
+- A self-hosted Teleport cluster in which the Auth Service and Proxy Service run
+  on Linux servers.
+
+  If you are running more than one Auth Service instance, you must reduce the
+  size of the Auth Service instance pool to one in order to perform an upgrade.
+
+Teleport supports automatic agent upgrades for systemd-based Linux distributions
+using `apt`, `yum`, and `zypper` package managers. The [Automatic Update
+Architecture](../architecture/agent-update-management.mdx) guide describes how
+agent updating works. Automatic agent upgrades require:
+
+- A Teleport cluster running version 13.0 or above.
+
+- At least one Teleport Enterprise agent, started via systemd on a distribution
+  using the `apt` or `yum` package managers.
+
+- A **version server** in your infrastructure that agents can query in order to
+  upgrade themselves. Read [Set up Automatic Upgrade
+  Infrastructure](./self-hosted-automatic-agent-updates.mdx) for how to deploy a
+  version server.
+
+  If your environment does not support automatic upgrades, you can follow the
+  same instructions to upgrade agents that we use to upgrade the Auth Service
+  and Proxy Service in this guide.
+
+## Step 1/2. Upgrade the Auth Service and Proxy Service
+
+Complete the following steps on all servers that run the Auth Service and Proxy
+Service:
+
+1. Install the latest Teleport version on the host.
+
+   (!docs/pages/includes/install-linux-self-hosted.mdx!)
+
+1. Confirm that the version of the `teleport` binary is the one you expect:
+
+   ```code
+   $ teleport version
+   ```
+
+1. Now that you have installed a more recent `teleport` binary on your Auth
+   Service and Proxy Service servers, restart Teleport on these servers to run
+   the new version.
+
+   (!docs/pages/includes/start-teleport.mdx!)
+
+## Step 2/2. Enroll agents in automatic upgrades
```

Comment:
Same comment than for self-hosted kube, I'm not sure automatic upgrades should be the default path for self-hosted.